File: Java_files/Step_4/Remainder_3052.java
package Step_4;
import java.util.Scanner;

public class Remainder_3052 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int [] numArr = new int[42];
        int i, a, tmp, count = 0;
        for(i = 0; i < 10; i++){
            a = sc.nextInt();
            tmp = a % 42;
            numArr[tmp]++;
        }
        for(i = 0; i < 42; i++){
            if(numArr[i] != 0)
                count++;
        }
        System.out.println(count);
    }
}

File: Bronze/Bronze_2/C_files/Code/Remainder_3052.c
#include <stdio.h>

int main()
{
	int numArr[42] = { 0, }, i, a, tmp, count = 0;
    for(i = 0; i < 10; i++){
        scanf("%d", &a);
        tmp = a % 42;
        numArr[tmp]++;
    }

    for(i = 0; i < 42; i++){
        if(numArr[i] != 0)
            count++;
    }
    printf("%d", count);
    return 0;
}